### PR TITLE
{lib}[GCCcore/12.3.0] tornado v6.3.2 w/ Python 3.11.3

### PR DESCRIPTION
**Pull request by surak**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by surak** [comment]
Tornado is necessary for Bokeh and iPython, and by Jupyter. It's better to have a general version for the future than conflicting ones inside other packages. I will add the new bokeh and ipython in time.

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/dd7c4c5c3f28781e70ce592302d59732 for a full test report.

File: easybuild/easyconfigs/t/tornado/tornado-timeouts.patch [hunk 1]
```diff
@@ -0,0 +1,15 @@
+diff -Naur tornado.orig/tornado-6.1/tornado/httpclient.py tornado/tornado-6.1/tornado/httpclient.py
```
Comment (casparvl) [inline easybuild/easyconfigs/t/tornado/tornado-timeouts.patch:1]: Can you put a comment in the past to indicate what it fixes? What was the error, is there an upstream issue...?

**Review by casparvl — CHANGES_REQUESTED** [review]

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Review by casparvl — APPROVED** [review]
Lgtm

**Comment by casparvl** [comment]
Going in, thanks @surak!

**Pull request merged**